openfisca_france/model/prestations/aide_mobilite.py

```diff
@@ -0,0 +1,377 @@
+    label = "Distance en kilomètres entre le lieu de l’entretien d’embauche, la reprise d’emploi, la formation, la prestation d’accompagnement, " \
+            "l’immersion professionnelle (PMSMP), le concours public ou l’examen certifiant et le lieu de résidence du demandeur d'emploi"
+    definition_period = MONTH
+    set_input = set_input_dispatch_by_period
+
+
+class nombre_allers_retours(Variable):
+    entity = Individu
+    value_type = float
+    reference = "http://www.bo-pole-emploi.org/bulletinsofficiels/deliberation-n-2021-42-du-8-juin-2021-bope-n2021-43.html?type=dossiers/2021/bope-n-2021-043-du-11-juin-2021"
+    label = "Nombre d'aller/retour pour le calcul de l'aide à la mobilité de Pôle emploi - AMOB"
+    definition_period = MONTH
+    set_input = set_input_dispatch_by_period
+
+
+class aide_mobilite_duree_trajet(Variable):
+    entity = Individu
+    value_type = float
+    reference = "http://www.bo-pole-emploi.org/bulletinsofficiels/deliberation-n-2021-42-du-8-juin-2021-bope-n2021-43.html?type=dossiers/2021/bope-n-2021-043-du-11-juin-2021"
+    label = "Durée en minutes entre le lieu de l’entretien d’embauche, la reprise d’emploi, la formation, la prestation d’accompagnement, " \
+            "l’immersion professionnelle (PMSMP), le concours public ou l’examen certifiant et le lieu de résidence du demandeur d'emploi"
+    definition_period = MONTH
+    set_input = set_input_dispatch_by_period
+
+
+class nuitees(Variable):
+    entity = Individu
+    value_type = int
+    reference = [
+        "http://www.bo-pole-emploi.org/bulletinsofficiels/deliberation-n-2021-42-du-8-juin-2021-bope-n2021-43.html?type=dossiers/2021/bope-n-2021-043-du-11-juin-2021"
+        ]
+    label = "Nombre de nuitées pour le calcul de l'aide à la mobilité de Pôle emploi - AMOB"
+    definition_period = MONTH
+    set_input = set_input_dispatch_by_period
+
+
+class repas(Variable):
+    entity = Individu
+    value_type = int
+    reference = [
+        "http://www.bo-pole-emploi.org/bulletinsofficiels/deliberation-n-2021-42-du-8-juin-2021-bope-n2021-43.html?type=dossiers/2021/bope-n-2021-043-du-11-juin-2021"
+        ]
+    label = "Nombre de repas pour le calcul de l'aide à la mobilité de Pôle emploi - AMOB"
+    definition_period = MONTH
+    set_input = set_input_dispatch_by_period
+
+
+class ContexteActivitePoleEmploi(Enum):
+    __order__ = 'indetermine formation reprise_emploi recherche_emploi'  # Needed to preserve the enum order in Python 2
+    indetermine = "Indeterminé"
+    formation = "En formation"
+    reprise_emploi = "En reprise d'emploi"
+    recherche_emploi = "En recherche d'emploi"
+
+
+class contexte_activite_pole_emploi(Variable):
+    value_type = Enum
+    possible_values = ContexteActivitePoleEmploi
+    default_value = ContexteActivitePoleEmploi.indetermine
+    entity = Individu
+    label = "Les différents contextes d'activité pour le calcul de l'aide à la mobilité de Pôle Emploi - AMOB "
```

Comment:
N'est-ce pas valable hors AMOB ? Définir un libellé qui désigne le sens d'une valeur unique ?
```suggestion
    label = "Le contexte d'activité au sens de Pôle Emploi"
```